src/App.tsx

```diff
@@ -9,32 +11,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab): void => {
+    const newTab = tabs.find(link => link.id === tab.id) || tabs[0];
+    const ind = tabs.indexOf(newTab);
```

Comment:
Why do you need this ?
We could just pass to `onTabSelected` function selectedTab

Reply:
changed